features/rhelah/atomic/rpm-ostree_bad_ca_cert.feature

```diff
@@ -0,0 +1,15 @@
+Feature: tests whether RH CA pinning works for rpm-ostree
+    It is critical for security that we're "pinned" to the RHN CA cert. If another server certificate is presented we should error out with something like Unacceptable TLS certificate.
+
+Scenario: 0. Host provisioned and subscribed
+    When "all" host is auto-subscribed to "prod"
+    Then subscription status is ok on "all"
+    and "1" entitlement is consumed on "all"
+
+Scenario: 1. Prepare mitmproxy
+    Given "git" are already installed on "rhel7"
+    and checkout the git repo "https://github.com/ljozsa/tools.git" on "rhel7"
+    and run command "command /root/tools/mitmproxy_install.sh" on "rhel7"
+
+Scenario: 2. Direct all https traffic through mitmproxy.
+    Then Run atomic host upgrade on "rhelah" using the mitmproxy on "rhel7"
```

Comment:
I suppose this is fine in practice, but it's not very "behave-like". Since these two scenarios are actually testing a single "behaviour" (that we get a certain error if the certificate is invalid), I would suggest grouping them under the same scenario. E.g.:

```gherkin
Scenario: Running through a proxy should give a TLS error
  Given "git" is already installed on "rhel7"
   When checkout the git repo "https://github.com/ljozsa/tools.git" on "rhel7"
    And run command "command /root/tools/mitmproxy_install.sh" on "rhel7"
   Then Run atomic host upgrade on "rhelah" using the mitmproxy on "rhel7"
```

Also, maybe rename the last step to clarify the expected outcome, e.g. `Then atomic host upgrade on "rhelah" using the proxy on "rhel7" should fail`.

I don't know how strictly we're adhering to behave testing, so maybe this is fine as it is (anyone want to chime in? @miabbott?).